diff --git a/loader/wine.inf.in b/loader/wine.inf.in
index aece31f8dc1..e283b2fa5e4 100644
--- a/loader/wine.inf.in
+++ b/loader/wine.inf.in
@@ -396,9 +396,9 @@ HKLM,%CurrentVersion%,"ProgramFilesDir (x86)",,"%16426%"
 HKLM,%CurrentVersion%,"CommonFilesDir (x86)",,"%16428%"
 HKLM,%CurrentVersion%,"ProgramFilesDir (Arm)",,"C:\Program Files (Arm)"
 HKLM,%CurrentVersion%,"CommonFilesDir (Arm)",,"C:\Program Files (Arm)\Common Files"
-HKLM,Software\Microsoft\Wow64\amd64,,2,"xtajit64.dll"
+HKLM,Software\Microsoft\Wow64\amd64,,2,"libarm64ecfex.dll"
 HKLM,Software\Microsoft\Wow64\arm,,2,"wowarmhw.dll"
-HKLM,Software\Microsoft\Wow64\x86,,2,"xtajit.dll"
+HKLM,Software\Microsoft\Wow64\x86,,2,"libwow64fex.dll"
 
 [CurrentVersionWow64.ntx86]
 HKLM,%CurrentVersion%,"ProgramFilesDir (x86)",,"%16422%"
@@ -492,8 +492,8 @@ HKLM,%Control%\Session Manager\KnownDLLs,"wldap32",,"wldap32.dll"
 HKLM,%Control%\Session Manager\KnownDLLs,"wow64",,"wow64.dll"
 HKLM,%Control%\Session Manager\KnownDLLs,"wow64win",,"wow64win.dll"
 HKLM,%Control%\Session Manager\KnownDLLs,"ws2_32",,"ws2_32.dll"
-HKLM,%Control%\Session Manager\KnownDLLs,"xtajit",,"xtajit.dll"
-HKLM,%Control%\Session Manager\KnownDLLs,"xtajit64",,"xtajit64.dll"
+HKLM,%Control%\Session Manager\KnownDLLs,"libwow64fex",,"libwow64fex.dll"
+HKLM,%Control%\Session Manager\KnownDLLs,"libarm64ecfex",,"libarm64ecfex.dll"
 HKLM,%Control%\Session Manager\Memory Management,PagingFiles,,"%24%\pagefile.sys 27 77"
 HKLM,%Control%\Session Manager\Memory Management,WriteWatch,0x00040002,1
 HKLM,%Control%\Session Manager\Memory Management\PrefetchParameters,BootId,0x10003,1
